src/test/AaveA3Caps26-02-2023-ByGuardian.t.sol

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import 'forge-std/Test.sol';
+
+import {IPoolConfigurator, ConfiguratorInputTypes, IACLManager} from 'aave-address-book/AaveV3.sol';
+import {AaveV3Avalanche} from 'aave-address-book/AaveAddressBook.sol';
+import {AaveV3AvaCapsSteward} from '../contracts/v3-ava-caps-26-02-2023/v3-ava-caps-26-02-2023.sol';
+import {AaveV3Helpers, ReserveConfig, ReserveTokens, IERC20} from './helpers/AaveV3Helpers.sol';
+
+contract AaveV3AvaChangeCapsByGuardian is Test {
+    using stdStorage for StdStorage;
+
+    address public constant GUARDIAN_AVALANCHE =
+        AaveV3Avalanche.ACL_ADMIN;
+
+    string public constant DAISymbol = 'DAI.e';
+    uint256 public constant DAI_SUPPLY_CAP = 30_000_000;
+    uint256 public constant DAI_BORROW_CAP = 20_000_000;
+
+    string public constant FRAXSymbol = 'FRAX';
+    uint256 public constant FRAX_SUPPLY_CAP = 1_500_000;
+    uint256 public constant FRAX_BORROW_CAP = 1_000_000;
+
+    string public constant MAISymbol = 'MAI';
+    uint256 public constant MAI_SUPPLY_CAP = 700_000;
+    uint256 public constant MAI_BORROW_CAP = 460_000;
+
+    string public constant USDCSymbol = 'USDC';
+    uint256 public constant USDC_SUPPLY_CAP = 250_000_000;
+    uint256 public constant USDC_BORROW_CAP = 175_000_000;
+
+    string public constant USDTSymbol = 'USDt';
+    uint256 public constant USDT_SUPPLY_CAP = 200_000_000;
+    uint256 public constant USDT_BORROW_CAP = 140_000_000;
+
+    string public constant AAVEeSymbol = 'AAVE.e';
+    uint256 public constant AAVEe_SUPPLY_CAP = 5_800;
+
+    string public constant LINKSymbol = 'LINK.e';
+    uint256 public constant LINKe_SUPPLY_CAP = 440_000;
+
+    string public constant BTCBSymbol = 'BTC.b';
```

Comment:
btcb,wbtc and weth seem to be not covered in the test actually

Reply:
added